src/components/ui/QuantitySelector/QuantitySelector.stories.mdx

```diff
@@ -0,0 +1,176 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs'
+
+import QuantitySelector from '.'
+import { TokenTable, TokenRow, TokenDivider } from 'src/../.storybook/components'
+
+<Meta
+  title="Molecules/QuantitySelector"
+  component={QuantitySelector}
+  argTypes={{
+    max: {
+      type: {
+        name: 'number',
+      },
+      defaultValue: 10,
+      description: "The maximum value the quantity selector can receive"
```

Comment:
Could these descriptions be on `QuantitySelector.txs` directly? [See Toggle.tsx](https://github.com/vtex-sites/gatsby.store/blob/main/src/components/ui/Toggle/Toggle.tsx) as reference.